commands/team.go

```diff
@@ -319,3 +336,80 @@ func removeDuplicatesAndSortTeams(teams []*model.Team) []*model.Team {
 	})
 	return result
 }
+
+func renameTeamCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	if len(args) == 0 || args[0] == "" {
+		return errors.New("Error: requires at least 1 arg(s), only received 0")
+	}
+
+	oldTeamName := args[0]
+	newDisplayName, _ := cmd.Flags().GetString("display_name")
+	newTeamName, _ := cmd.Flags().GetString("name")
+
+	// If both flags are absent, abort!
+	if newTeamName == "" && newDisplayName == "" {
+		return errors.New("Require atleast one flag to rename team, either 'name' or 'display_name'")
+	}
+
+	team := getTeamFromTeamArg(c, oldTeamName)
+	if team == nil {
+		return errors.New("Unable to find team '" + oldTeamName + "', to see the all teams try 'team list' command")
+	}
+
+	// Check for name uniqueness
+	if newTeamName != "" && newTeamName == team.Name {
+		return errors.New("Entered name is the current name for " + team.Name + " , either remove the flag or updage to new value")
+	}
+
+	// Check for display name uniqueness
+	if newDisplayName != "" && newDisplayName == team.DisplayName {
+		return errors.New("Entered display name is the current display name for " + team.Name + " , either remove the flag or updage to new value")
+	}
```

Comment:
Same here. This is a specially valid case for scripts and sequential updates; if the team name or display name to update are already set, it is fine to send the request and return an ok to the user as they will find the name and display name that they expect